### ISSUE TITLE
Selective import works strange

### ISSUE THREAD

**Issue by Terisback** [vlang/v#7164]
**V version:** V 0.1.30 853dcc3
**OS:** linux, Ubuntu 20.04.1 LTS (VM) (WSL)

**What did you do?**
```v
// ./client/client.v
module client

import crypto.sha1

pub fn new(token string) {
	hash := sha1.hexhash(token)
	println('init client with token: $hash')
}

// ./main.v
module main

import client { new }

fn main() {
	new("WASSUP")
}
```

**What did you expect to see?**
```
init client with token: $some-hash
```
 
**What did you see instead?**
```v
/home/teris/v/vlib/crypto/sha1/sha1.v:136:11: error: too few arguments in call to `client.new` (0 instead of 1)
  134 | // Sum returns the SHA-1 checksum of the data.
  135 | pub fn sum(data []byte) []byte {
  136 |     mut d := new()
      |              ~~~~~
  137 |     d.write(data)
  138 |     return d.checksum()
```

**Comment by Terisback** [comment]
The main thing is that if imported module imports module with functions with same name it goes bruh

**Comment by joe-conigliaro** [comment]
This issue is fixed in https://github.com/vlang/v/pull/7182

**Comment by spytheman** [comment]
can confirm - closed by #7182

**Comment by joe-conigliaro** [comment]
Thanks @spytheman
